### ISSUE TITLE
Review fillCellExample

### ISSUE THREAD

**Issue by avancinirodrigo** [TerraME/terrame#1635]
The `fillCellExample.lua` is deprecated, it needs a review or even to remove it.

**Comment by pedro-andrade-inpe** [comment]
Remove this example and add a file named `amazonia-postgis.lua`, which executes the same code of `amazonia.lua`, but saves the cellular layer into a postgis database.

**Comment by avancinirodrigo** [comment]
Do this after #815.

**Comment by pedro-andrade-inpe** [comment]
The updated version of this script is shown below. It has some problems related in the script itself, but it also stops with an error:

```
C:\test>terrame -color -package terralib -project postgis-amazonia
Creating 'C:/test/amazonia.shp' with resolution 50000
Executing operation 'coverage' using layer 'prodes'
Could not execute the script properly: C:/TerraME/bin/packages/terralib/data\postgis-amazonia.lua:84: Error: Could not execute the sql statement due to the following error: ERROR:  relation "cells_temp" already exists.
```

```lua
-- @example Creates and fills amazonia database in PostGIS DataBase Management System.

import("terralib")

amazonia = Project{
	file = "amazonia.tview",
	clean = true,
	title = "Amazonia database",
	ports = filePath("amazonia-ports.shp", "terralib"),
	roads = filePath("amazonia-roads.shp", "terralib"),
	limit = filePath("amazonia-limit.shp", "terralib")
}

prodes = Layer{
	name = "prodes",
	project = amazonia,
	srid = 29191,
	file = filePath("amazonia-prodes.tif", "terralib")
}

protected = Layer{
	name = "protected",
	project = amazonia,
	srid = 29191,
	file = filePath("amazonia-indigenous.shp", "terralib")
}

if not getConfig() then
	customError("You should provide connection information in a config.lua file.")
end

-- the lines below related to terralib should not be necessary. Layer{} call
-- below shoud handle that as it uses 'clean = true'.
local terralib = TerraLib{}
terralib:dropPgTable{
	type = "POSTGIS",
	port = 5432,
	user = getConfig().user or "postgres",
	password = getConfig().password,
	database = "amazonia",
	table = "amazonia", 
	host = "localhost",
	encoding = "CP1252"
}

amazoniaCells = Layer{
	project = amazonia,
	file = "amazonia.shp",
	clean = true,
	input = "limit",
	name = "cells",
	resolution = 50000,
	source = "postgis",
	user = getConfig().user or "postgres", -- this should be only "user = getConfig().user" - "postgres" should be the default value for user
	password = getConfig().password,
	database = "amazonia",
	table = "amazonia"
}

amazoniaCells:fill{
	operation = "coverage",
	layer = "prodes",
	attribute = "prodes"
}

amazoniaCells:fill{
	operation = "distance",
	layer = "roads",
	attribute = "distroads"
}

amazoniaCells:fill{
	operation = "distance",
	layer = "ports",
	attribute = "distports"
}

amazoniaCells:fill{
	operation = "area",
	layer = "protected",
	attribute = "protected"
}


cs = CellularSpace{
	project = amazonia,
	layer = "cells",
	as = {
		forest = "prodes_208",
		deforestation = "prodes_10"
	}
}

Map{
	target = cs,
	select = "distroads",
	slices = 10,
	invert = true,
	color = "YlOrBr"
}

Map{
	target = cs,
	select = "distports",
	slices = 10,
	invert = true,
	color = "YlOrBr"
}

Map{
	target = cs,
	select = "protected",
	slices = 10,
	invert = true,
	color = "PuBu"
}

Map{
	target = cs,
	select = "deforestation",
	slices = 10,
	invert = true,
	color = "YlGn"
}
```

**Comment by avancinirodrigo** [comment]
The problem with `_temp` have been resolved in #1651.

Now, there is a problem with `-projects` for postgis:
``` bash
Processing 'data/postgis-amazonia.lua'
Creating 'data/postgis-amazonia.tview'
Could not execute the script properly: ...eveloper/terrame/build-cmake/install/bin/lua/project.lua:196: attempt to index a nil value (local 'mfile')
Removing file 'data/postgis-amazonia.tview'
```
